src/compute/mod.rs

```diff
@@ -23,33 +24,6 @@ pub use self::grid::CssGridAlgorithm;
 #[cfg(any(feature = "debug", feature = "profile"))]
 use crate::util::debug::NODE_LOGGER;
 
-/// Updates the stored layout of the provided `node` and its children
```

Comment:
We currently have a nice grouping of code where all of the computation happens in the `compute` module. I would like to keep that. Please can you move these function back into the `compute` module. If you want to reduce the number of `#[cfg(feature = "taffy")]`s then perhaps you could create a sub-module of `compute` to put these functions in.